examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -31,25 +31,42 @@
 import java.util.logging.Logger;
 import javax.net.ssl.SSLException;
 
+// TODO: change flags to use com.google.common.flags
+/*
+import com.google.common.flags.Flag;
+import com.google.common.flags.FlagSpec;
+import com.google.common.flags.Flags;
+*/
+
+
+
 /**
  * A simple client that requests a greeting from the {@link HelloWorldServerTls} with TLS.
```

Comment:
Please phrase this as "Requests a greeting from...". Also please mention that it uses TLS 1.3, and you don't need to mention the specific server instance (since it could connect to a different instance if it wanted to).